doc/src/manual/workflow-tips.md

```diff
@@ -64,8 +64,13 @@ line. A common pattern includes the following elements:
 
 ## Browser-based workflow
 
-It is also possible to interact with a Julia REPL in the browser via [IJulia](https://github.com/JuliaLang/IJulia.jl).
-See the package home for details.
+Here are a few ways to interact with the Julia REPL in a browser:
+
+  * One way is via [IJulia](https://github.com/JuliaLang/IJulia.jl). 
+    See the package home for details.
+
+  * Another way is via [Pluto](https://github.com/fonsp/Pluto.jl) notebooks. 
+    See the [wiki](https://github.com/fonsp/Pluto.jl/wiki) for details.
```

Comment:
Fons is actually working on a new website so this comment will become outdated pretty soon. Maybe remove both suggestion for details and just link to the core GitHub page?

Reply:
@rikhuijzer done, is it fine now?